examples/main.tf

```diff
@@ -344,7 +344,8 @@ resource "instaclustr_cluster" "example-postgresql" {
     options = {
       postgresql_node_count = 1,
       client_encryption = true,
-      replication_mode = "SYNCHRONOUS"
+      replication_mode = "SYNCHRONOUS",
+      synchronous_mode_strict = true
```

Comment:
[MINOR]
Would be better to have as false which is the default for API.

Reply:
Changed to false